File: array.c
//Creating a dynamic array accepting input from user
#include <stdio.h>
#include <stdlib.h>
int main()
{
    int i,j,m,n;
    int *arr;
    printf("Enter the array size");
    scanf("%d", &n);
    arr=(int*)malloc(n*sizeof(int));
    for(i=1;i<=n;i++)
    {
        printf("Enter the Element %d", i);
        scanf("%d", &arr[i]);
    }
    for(j=1;j<=n;j++){
        printf("%d\t", arr[j]);   
    }
}

File: array.java
//Creating a dynamic that accepts input from user
import java.util.Scanner;
public class Main
{
    public static void main(String[] args)
    {
        int n;
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the number of Elements you want to store: ");
    
        n=sc.nextInt();
        int[]array=new int[10];
        System.out.println("Enter the Elements: ");
        for(int i=0;i<n;i++)
        {
            array[i]=sc.nextInt();
        }
        System.out.println("Array Elements are: ");
        for(int i=0;i<n;i++)
        {
            System.out.println(array[i]);
        }
    }
}

